index.bs

```diff
@@ -8846,8 +8846,10 @@ specified sequence of user input actions.
       )
 
       input.AngleProperties = (
-        ? altitudeAngle: float .default 0.0,
-        ? azimuthAngle: float .default 0.0,
+        ; 0 .. 2 * Math.PI
+        ? altitudeAngle: (0.0..6.283185307179586) .default 0.0,
+        ; 0 .. Math.PI / 2
+        ? azimuthAngle: (0.0..1.5707963267948966) .default 0.0,
```

Comment:
I think that's the opposite? azimuthAngle is between 0 and 2π and altitudeAngle is between 0 and π/2

Reply:
oh right, overlooked that.

Reply:
Indeed!